include/seqan3/alphabet/quality/qualified.hpp

```diff
@@ -165,6 +165,270 @@ struct qualified :
+    {
+        return std::get<0>(*this) == rhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(rhs) == \p lhs, `false` otherwise.
+    friend constexpr bool operator==(sequence_alphabet_t const & lhs,
+                                     qualified<sequence_alphabet_t, quality_alphabet_t> const & rhs)
+    {
+        return rhs == lhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(*this) != \p rhs, `false` otherwise.
+    constexpr bool operator!=(sequence_alphabet_t const & rhs) const noexcept
+    {
+        return std::get<0>(*this) != rhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(rhs) != \p lhs, `false` otherwise.
+    friend constexpr bool operator!=(sequence_alphabet_t const & lhs,
+                                     qualified<sequence_alphabet_t, quality_alphabet_t> const & rhs)
+    {
+        return rhs != lhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(*this) < \p rhs, `false` otherwise.
+    constexpr bool operator<(sequence_alphabet_t const & rhs) const noexcept
+    {
+        return std::get<0>(*this) < rhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(rhs) < \p lhs, `false` otherwise.
+    friend constexpr bool operator<(sequence_alphabet_t const & lhs,
+                                    qualified<sequence_alphabet_t, quality_alphabet_t> const & rhs)
+    {
+        return rhs < lhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(*this) > \p rhs, `false` otherwise.
+    constexpr bool operator>(sequence_alphabet_t const & rhs) const noexcept
+    {
+        return std::get<0>(*this) > rhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(rhs) > \p lhs, `false` otherwise.
+    friend constexpr bool operator>(sequence_alphabet_t const & lhs,
+                                    qualified<sequence_alphabet_t, quality_alphabet_t> const & rhs)
+    {
+        return rhs > lhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(*this) <= \p rhs, `false` otherwise.
+    constexpr bool operator<=(sequence_alphabet_t const & rhs) const noexcept
+    {
+        return std::get<0>(*this) <= rhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(rhs) <= \p lhs, `false` otherwise.
+    friend constexpr bool operator<=(sequence_alphabet_t const & lhs,
+                                     qualified<sequence_alphabet_t, quality_alphabet_t> const & rhs)
+    {
+        return rhs <= lhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(*this) >= \p rhs, `false` otherwise.
+    constexpr bool operator>=(sequence_alphabet_t const & rhs) const noexcept
+    {
+        return std::get<0>(*this) >= rhs;
+    }
+
+    //!\brief Returns `true` if \p std::get<0>(rhs) >= \p lhs, `false` otherwise.
+    friend constexpr bool operator>=(sequence_alphabet_t const & lhs,
+                                     qualified<sequence_alphabet_t, quality_alphabet_t> const & rhs)
+    {
+        return rhs >= lhs;
+    }
+    //!\}
```

Comment:
I usually write the non-firends in one block and then the friends, but I guess it you could argue for either style...